jsonify_stored/readme/ROADMAP.rst

```diff
@@ -0,0 +1,5 @@
+- Add tests!!!
```

Comment:
mention that this work originated from connector_search_engine and that module will be refactored on top of this in the future